include/seqan/misc/bit_twiddling.h

```diff
@@ -272,7 +272,7 @@ popCount(TWord word)
 // intrinsics __popcnt16, __popcnt, and __popcnt64 for 16, 32, and 64 bit words.
 
 // MSVC >= 2008, has intrinsic
-#if defined(_MSC_VER)
+#if defined(STDLIB_VS)
```

Comment:
since these are compiler builtins, maybe use `defined(COMPILER_MSVC) || defined(COMPILER_WINTEL)` instead? I guess clang/c2 can use the regular builtins...